src/main/webapp/json/starter-templates.json

```diff
@@ -159,6 +159,41 @@
           "hasWidgetURL": false
         }
       }
+    },
+    {
+      "entry": {
+        "layoutObject": {
+          "id": 7,
+          "url": "//www.example.com",
```

Comment:
What's with the `"//` ? I'm not familiar with this convention, potentially about to learn something here...

Reply:
It sets the protocol on the link based on the current protocol. So, if your accessing the current site by `https`, the link will appear as `https://www.example.com`.

Using it for these links isn't really the most correct use case. The correct use case is for loading resources (scripts, images) on the page all with the same protocol, so that you don't get an unsecure content warning when your site is served by https.